src/golang/lib/functional_primitives/functional_slice/slice.go

```diff
@@ -0,0 +1,17 @@
+package functional_slice
```

Comment:
@saurav-c what do you suggest for naming? The main issue is `maps` and `slice` appear to be reserved. As a result I couldn't do `maps / slices` or `map/slice` :(

Reply:
Maybe just `lib/functional` and then we can `map.go` and `slice.go` inside.

Reply:
Actually somehow `maps` and `slices` working, and we can proceed with these names